src/js/api/auth/login.test.js

```diff
@@ -0,0 +1,44 @@
+import { login } from "./login.js";
+//import {save} from "../../storage/index.js";
+
+class LocalStorageMock {
+  constructor() {
+    this.store = {};
+  }
+
+  clear() {
+    this.store = {};
+  }
+
+  getItem(key) {
+    return this.store[key] || null;
+  }
+
+  setItem(key, value) {
+    this.store[key] = value.toString();
+  }
+
+  removeItem(key) {
+    delete this.store[key];
+  }
+}
+
+global.localStorage = new LocalStorageMock();
+
+describe("login", () => {
+  it("should save the token and profile to storage", async () => {
+    global.fetch = jest.fn().mockImplementation(() => {
+      return Promise.resolve({
+        ok: true,
+        json: () => {
+          return Promise.resolve({
+            accessToken: "token",
+          });
+        },
+      });
+    });
+    await login("email", "password");
+    localStorage.setItem("token", "token");
+    expect(localStorage.getItem("token")).toBe("token");
+  });
```

Comment:
Well done on the valid test case. However, I think a test case is not enough to cover the testing of the login functionality. You add more test cases such as when invalid credentials are supplied. The same thing applies to the rest of your tests.

Reply:
I've added a test to create.test.js and login.test.js to test that it should throw and error if the response is not OK